src/commands/doctor.rs

```diff
@@ -0,0 +1,96 @@
+use std::{fmt::Debug, path::PathBuf};
+
+use anyhow::Result;
+use clap::Parser;
+use dialoguer::{console::Emoji, Select};
+use futures::FutureExt;
+use spin_doctor::Diagnosis;
+
+#[derive(Parser, Debug)]
+#[clap(hide = true, about = "Run the Spin Doctor")]
+pub struct DoctorCommand {
+    #[clap(short = 'f', long, default_value = "spin.toml")]
+    file: PathBuf,
+}
+
+impl DoctorCommand {
+    pub async fn run(self) -> Result<()> {
+        eprintln!("{icon}The Spin Doctor is in.", icon = Emoji("📟 ", ""));
+        eprintln!("{icon}Checking {:?}...", self.file, icon = Emoji("🩺 ", ""));
```

Comment:
I default to `Debug` for string-like things in general because it can reveal surprising characters to aid debugging. I agree that for this particular case `display` probably makes more sense.